bicep/networkwatcherflowlog.bicep

```diff
@@ -12,6 +12,11 @@ param workspaceId string = ''
 param workspaceResourceId string = ''
 param workspaceRegion string = resourceGroup().location
 
+resource networkWatcher 'Microsoft.Network/networkWatchers@2022-01-01' = {
```

Comment:
Do we need to worry about the scope that this is run at?
It definitely creates in the right resource group?